CMP1903M Assessment 1 Base Code/Input.cs

```diff
@@ -8,27 +8,74 @@ namespace CMP1903M_Assessment_1_Base_Code
 {
     public class Input
     {
-        //Handles the text input for Assessment 1
-        string text = "nothing";
-        
+        //Creating default variables
+        public string text = "nothing";
+        public string? path;
+
         //Method: manualTextInput
         //Arguments: none
         //Returns: string
         //Gets text input from the keyboard
-        public string manualTextInput()
+        public void manualTextInput()
         {
-
-            return text;
+            Console.WriteLine("Please input the text you wish to analyse.");
+            text = Console.ReadLine();
+            if (text == null)
+            {
+                text = "";
+            }
         }
 
         //Method: fileTextInput
         //Arguments: string (the file path)
         //Returns: string
         //Gets text input from a .txt file
-        public string fileTextInput(string fileName)
+        public void fileTextInput(string fileName)
+        {
+            try
+            {
+                text = File.ReadAllText(fileName);
+            }
+            catch (FileNotFoundException e) //Terminate the program if the file is not found

```

Comment:
No validation for file location, instead of terminating program possibly utilising a loop system to prompt the user to enter another location but otherwise great